lib/ide/src/entry_point.rs

```diff
@@ -0,0 +1,7 @@
+#![allow(missing_docs)]
+
+use super::view::project_view::ProjectView;
+
+pub fn entry_point() {
+    ProjectView::new().forget();
+}
```

Comment:
What is this entry point about? What is the responsibility of that? I feel it is one of the most important parts of this PR and it needs solid documentaiton.